.pre-commit-config.yaml

```diff
@@ -22,12 +22,12 @@ repos:
     hooks:
     -   id: seed-isort-config
         args: [--application-directories, '.:src']
--   repo: https://github.com/pre-commit/mirrors-isort
-    rev: v5.6.1
+-   repo: https://github.com/PyCQA/isort
+    rev: 5.6.4
```

Comment:
https://github.com/pre-commit/mirrors-isort is deprecated and the main isort repo should be used instead.

Reply:
No problems. I’ll eventually give it a go.